docs/tutorials/segmentation/index.md

```diff
@@ -1,5 +1,5 @@
 # Segmentation
 
-This section contains tutorials for segmentation.
+This section contains tutorials explaining use-cases for segmentation [labelling](../howtos/layers/labels). 
```

Comment:
```suggestion
This section contains tutorials explaining use-cases for segmentation [labeling](../howtos/layers/labels). 
```

Though I appreciate the Aussie spelling, we have decided to use US spelling everywhere 😂 

Thanks @tdimino!

Reply:
As with others, like the link, not the extra words